src/frontend/style/cart.css

```diff
@@ -0,0 +1,159 @@
+.cart {
+    max-width: 1200px;
+    margin: 2rem auto;
+}
+
+.cart-grid {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+}
+
+.horizointal-card.bat-card img {
+    width: 120px;
+    height: 130px;
+    object-position: top;
+    object-fit: cover;
+}
+
+.horizointal-card.bat-card {
+    border-bottom: 1px solid #e0e0e0;
+    border-right: 1px solid #e0e0e0;
+    flex-direction: row;
+    max-width: 100%;
+    overflow: unset;
+    gap: 0.5rem;
+}
+
+.cart-add-qnt {
+    display: flex;
+    gap: .35rem;
+    align-items: center;
+    margin: 0 1rem;
+}
+
+.cart-add-qnt span:nth-child(2) {
+    border: 1px solid var(--grey);
+    height: 1.5rem;
+    width: 3rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
you can create flex utility classes later on

Reply:
I've created those classes, forgot to use, Will update.